pajbot/apiwrappers/base.py

```diff
@@ -73,25 +73,34 @@ def join_base_and_endpoint(base, endpoint):
         else:
             return BaseAPI.join_base_and_string(base, endpoint)
 
-    def request(self, method, endpoint, params, headers, json=None, **request_options):
-        full_url = self.join_base_and_endpoint(self.base_url, endpoint)
+    def request(self, method, endpoint, params, headers, json=None, base_url=None, data=None, **request_options):
+        if base_url is None:
+            base_url = self.base_url
```

Comment:
The addition of the `base_url` parameter here stinks clear misuse of the apiwrappers base, if you need to access multiple api "groups", then, then multiple classes should be created.
See twitch apiwrappers for example (id and helix)